Cargo.toml

```diff
@@ -15,8 +15,8 @@ exclude = ["/.github/*", "/.travis.yml", "/appveyor.yml"]
 
 [dependencies]
 log = { version = "0.4", features = ["std"] }
-lazy_static = "1.2"
-rand = "0.7"
+lazy_static = "1.4"
```

Comment:
Why? I think cargo is able to select a latest version if necessary.

Reply:
done